pyIocApp/setup.c

```diff
@@ -135,7 +139,7 @@ static void pySetupReg(void)
     setupPyPath();
 
     if(PyRun_SimpleString("import devsup\n"
-                          "devsup._init(iocMain=True)\n"
+                          "devsup._init(iocMain=False)\n"
```

Comment:
Is this change necessary now that `_init()` does not use `NamedTemporaryFile`?

Reply:
The two are separate in _init(). I'm not too sure what was intended here, but I do not see any existing call of _init() that caused base to load, and this is still the case.

Reply:
base would of course have been loaded by the IOC start-up.
So I'm not sure whether loading it from _dbapi is actually required anyway.